templates/default/less/variables.less

```diff
@@ -122,6 +122,9 @@
 //
 //##
 @icon-font-path:            "../../Services/UICore/lib/bootstrap-3.2.0/fonts/";
+@icon-size-small: 20px;
+@icon-size-medium: 35px;
+@icon-size-large: 45px;
 
```

Comment:
@Amstutz Please see my last commit; is this the correct way? Thanks for your help.